server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java

```diff
@@ -999,6 +1002,12 @@ private TemplateVariables buildTemplateVariables(CreateFlowInstanceReq flowInsta
         return variables;
     }
 
+    private void setDefaultSchemaIfNeed(CreateFlowInstanceReq req, ConnectionConfig config) {
+        if (req.getTaskType() == TaskType.ASYNC && config.getDialectType() == DialectType.ORACLE) {
```

Comment:
Why we should set default schema for Oracle and specifical for ASYNC task?

Reply:
because oracle JDBC can not set default schema, when we create a new connection by JDBC, the default current schema is current user, so we need to set default schema to trigger `SwitchSchemaInitializer` to switch schema to database change schema.

Reply:
I have deleted this method, the database base change task will switch schema to database change schema without this method.